plugins/rapid7_insightvm/help.md

```diff
@@ -181,15 +181,15 @@ Example output:
 
 #### Get Asset Vulnerability Solutions
   
-Returns the highest-superceding rollup solutions for a list of vulnerabilities on an asset
+This action is used to returns the highest-superceding rollup solutions for a list of vulnerabilities on an asset
```

Comment:
Could be updated in the plugin spec to allow this description to read better 

`Return the highest-superceding rollup solutions for a list of vulnerabilities on an asset`